wp-content/themes/csisnuclearnetwork/inc/template-tags.php

```diff
@@ -225,23 +225,25 @@ function nuclearnetwork_authors_list_extended()
 		}
 
 		if (function_exists('coauthors_posts_links')) {
-			$authors = '<h2 class="section__heading">Authors</h2>';
+			$authors = '<h2 class="section__heading post__authors-heading">Authors</h2>';
+			$authors .= '<p class="text--italic post__authors-disclaimer">The views expressed above are the author’s and do not necessarily reflect those of the Center for Strategic and International Studies or the Project on Nuclear Issues.</p>';
 
 			foreach (get_coauthors() as $coauthor) {
 				$name = $coauthor->display_name;
-
-				$authors .= '<div class="post__authors-author"><h3 class="post__authors-author-name">' . $name . '</h3><p class="post__authors-author-bio">' . $name . ' ' . $coauthor->description . '</p>';
-
-				if ( $coauthor->website ) {
-					$authors .= '<a href="' . $coauthor->website . '" class="post__authors-author-link">Learn More ' . nuclearnetwork_get_svg( 'arrow-external' ) .'</a></div>';
+				$title = $coauthor->title;
```

Comment:
The title isn't appearing because it is an ACF, so you'll need to use `get_field()`. The the comment below for an example.

Reply:
Updated